lib/LoaderRunner.js

```diff
@@ -178,7 +178,7 @@ function iteratePitchingLoaders(options, loaderContext, callback) {
 			function(err) {
 				if(err) return callback(err);
 				var args = Array.prototype.slice.call(arguments, 1);
-				if(args.length > 0) {
+				if(args[0] !== undefined) {
```

Comment:
Could you check if all `args` are `undefined`?

i. e. `done(null, undefined, 1)` should break, `done(null, undefined, undefined)` not.